multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -309,6 +309,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Manhwa365", "https://manhwa365.com", "en", isNsfw = true),
         SingleLang("Manhwa68", "https://manhwa68.com", "en", isNsfw = true, overrideVersionCode = 3),
         SingleLang("ManhwaBookShelf", "https://manhwabookshelf.com", "en"),
+        SingleLang("ManhwaClan", "https://manhwaclan.com", "en", overrideVersionCode = 2),
```

Comment:
First versions should not include the `overrideVersionCode`.

```suggestion
        SingleLang("ManhwaClan", "https://manhwaclan.com", "en"),
```